src/main/java/com/jalasoft/selenium/leonardo/movies/Movie.java

```diff
@@ -0,0 +1,47 @@
+package com.jalasoft.selenium.leonardo.movies;
+
+/**
+ *
+ */
+public class Movie extends movieType {
+    public static final int CHILDRENS = 2;
+    public static final int REGULAR = 0;
+    public static final int NEW_RELEASE = 1;
```

Comment:
Do you still need to keep those constants ?
you need to take advantage of inheritance here.